weasyl/polecat.py

```diff
@@ -1,5 +1,7 @@
 """Polecats are twisted weasyls.
 
+from __future__ import absolute_import
```

Comment:
This will never get executed. Docstring begins at L1, ends at L6, as of this commit.